test/commands/endpoints/access/add.test.ts

```diff
@@ -8,8 +8,20 @@ describe('endpoints:access:add', () => {
     )
     .stdout()
     .stderr()
-    .command(['endpoints:access:add', 'postgres-123', '--account-ids', 'arn:aws:iam::123456789:root', '--app', 'myapp'])
+    .command(['endpoints:access:add', 'postgres-123', '--account-ids', '123456789:root', '--app', 'myapp'])
     .it('adds an account to the whitelist', ctx => {
       expect(ctx.stderr).to.contain('Adding account to the whitelist... done')
     })
+
+  test
+    .nock('https://postgres-api.heroku.com', api => api
+      .put('/private-link/v0/databases/postgres-123/whitelisted_accounts')
+      .reply(200, {})
+    )
+    .stdout()
+    .stderr()
+    .command(['endpoints:access:add', 'postgres-123', '--account-ids', '"123456789:resource1 123456789:resource2"', '--app', 'myapp'])
```

Comment:
I don't think you need quotes around the `--account-ids` flag value here because the quotes are for the *shell* not the CLI itself, and the `command` here is simulating what happens after your shell parses the command. Also shouldn't nock have an expectation on the payload in this test?